inbox/api/srv.py

```diff
@@ -130,4 +130,5 @@ def logout():
 
 
 app.register_blueprint(ns_api)
+app.register_blueprint(ns_api, url_prefix='/v2')
```

Comment:
...what is this for? cruft before we decided to go with gating rather than changing the URL?